github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml

```diff
@@ -41,6 +41,42 @@ periodics:
         resources:
           requests:
             memory: "200Mi"
+  - name: periodic-update-common-instancetypes-bundles-release-1.1
+    cron: 0 5 * * *
+    annotations:
+      testgrid-create-test-group: "false"
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    max_concurrency: 1
+    cluster: kubevirt-prow-control-plane
+    extra_refs:
+    - org: kubevirt
+      repo: kubevirt
+      base_ref: release-1.3
+    labels:
+      preset-github-credentials: "true"
+    spec:
+      containers:
+      - image: quay.io/kubevirtci/pr-creator:v20240305-cb764ec
+        env:
+          - name: COMMON_INSTANCETYPES_BRANCH
+            value: "release-1.0"
+          - name: KUBEVIRT_BRANCH
```

Comment:
I don't see SSP release-v0.21 here like the description suggests?

Reply:
Apologies I edited the commit message and forgot about the PR description. That release doesn't exist yet so we can't sync anything for now.